### ISSUE TITLE
Pgsql has gone away

### ISSUE THREAD

**Issue by pivnicek** [keboola/db-extractor-pgsql#41]
Receiving this occasional error
```
General error: 7 server closed the connection unexpectedly \tThis probably means the server terminated abnormally \tbefore or while processing the request. 
```
Seems to be coming from the same backend though, so may be db server settings specific

ref: https://keboola.slack.com/archives/C09U3R1J4/p1543481464025800

**Comment by pivnicek** [comment]
This isn't getting retried?

**Comment by ondrejhlavacek** [comment]
and multiple times during last night, the same project though

**Comment by odinuv** [comment]
i think the retry mechanism is broken - there is no delay

![image](https://user-images.githubusercontent.com/4319320/51315051-93737000-1a51-11e9-9a27-0ebe01753616.png)

**Comment by pivnicek** [comment]
should be resolved by 2.5.7
